src/core/control/Control.h

```diff
@@ -217,6 +217,7 @@ class Control:
     void appendNewPdfPages();
     void insertPage(const PageRef& page, size_t position, bool shouldScrollToPage = true);
     void deletePage();
+    void movePage(size_t oldPos, size_t newPos);
```

Comment:
This would benefit a comment (in particular stating that the page positions start at 0).